### DISCUSSION TITLE
CoPilot rewrites the whole code block in notebook instead of selection only

### DISCUSSION THREAD

**Discussion by mascaritas** [community/community#78538] [Copilot Conversations]
I find it very hard to control/restrict CoPilots Suggestion (ctrl+I on a selection of code) on only the selected section. 

What is a good practice to achieve "more surgical" suggestion when working with Jupyter Notebooks in VSCode?

Thanks for any hints

**Comment by foursixnine** [comment]
@mascaritas I wonder if this is similar to #83077
